utilities/pipelines/staticValidation/compliance/module.tests.ps1

```diff
@@ -192,6 +205,145 @@ Describe 'File/folder tests' -Tag 'Modules' {
                 $pathExisting | Should -Be $false -Because ('The module is not orphaned but owned by [{0}].' -f $relevantCSVRow.PrimaryModuleOwnerGHHandle)
             }
         }
+
+        # Changelogs are required for all (child-)modules that are published (having a version.json file)
+        It '[<moduleFolderName>] Module should contain a [` CHANGELOG.md `] file.' -TestCases ($moduleFolderTestCases | Where-Object { $_.moduleVersionExists }) {
+
+            param(
+                [string] $moduleFolderPath,
+                [bool] $moduleVersionExists
+            )
+
+            $changelogFilePath = Join-Path -Path $moduleFolderPath 'CHANGELOG.md'
+            $pathExisting = Test-Path $changelogFilePath
+            $pathExisting | Should -Be $true
+        }
+
+        It '[<moduleFolderName>] `CHANGELOG.md` must contain a section with content and versions are ordered descending.' -TestCases ($moduleFolderTestCases | Where-Object { $_.moduleVersionExists }) {
```

Comment:
This test looks very long. I'd suggest to keep it more readable by in case adding more functions to the helper modules. Also it may be advisable to divide it to only cover one specific test (e.g. have 2 test one to ensure each section has content and one to ensure versions are descending). We can retrieve changelog content in the before all section to avoid duplicating it